modules/ROOT/pages/_partials/identity-management-demo.adoc

```diff
@@ -3,13 +3,26 @@
 = {keycloak-service}
 
 // tag::idm-demo[]
-
 From the showcase app:
 
-. Press the *Authenticate* menu item. A login screen is displayed.
+[tabs]
```

Comment:
should we switch the tabs everywhere (over time)?

Reply:
yes, I think so